markdown/blog/2025-Q2-marketing-report.md

```diff
@@ -0,0 +1,291 @@
+
+- Drive high-quality CFP submissions and attendee diversity for Singapore, Munich, Lagos events
+
+- Deepen community engagement across LinkedIn, X, Slack, and newsletter
+
+- Promote the community members and expand content marketing via spotlight stories
+
+- Introduce the Webinar Initiative.
+
+## Strategy Alignment
+
+All Q2 activities were implemented to advance the approved [Community Marketing Strategy](https://github.com/asyncapi/community/blob/master/docs/030-project-vision-strategy-goals/2025_marketing_strategy.md). Below we map Q2 actions to each strategic goal, evidence from Q2, and recommended next steps.
+
+### Goal 1 — Encourage Adoption of AsyncAPI & EDA 
+
+**Q2 Activities**
+
+- AsyncAPI track at APIDays Singapore
+
+- The "Learn With AsyncAPI" webinar series documentation has been merged and the first session is on the 30th of September.
+
+- Conference workshops and hands-on sessions
+
+**Evidence**
+
+- Strong in-person attendance in Singapore
+- Webinar engagement and repurposed content
+- Newsletter open rate of 42%; 
+- AsyncAPI Singapore recap video (1.8k views).
+
+**Gap/Next steps**
+
+Multilingual documentation target (one translation by end of Q2) was planned in the strategy, work was but has not been initiated.
+Recommend prioritizing at least one translated guide, and preparing localized workshop materials for Lagos.
+
+### Goal 2 — Showcase Exceptional Talent
+
+**Q2 Activities**
+
+Community spotlights, newsletter features highlighting maintainers and first-time speakers, and a CFP drive focused on diverse representation.
+
+**Evidence**
+
+- CFP submissions: Munich — 63 (40% new speakers)
+- Lagos — 48 (100% Africa-based; 60% women-led).
+- Increased engagement on spotlight posts and positive community feedback.
+
+**Next steps**
+
+- Publish in-depth case studies and maintainership stories
+- Encourage the community members to make use of the holopin badges recognition program to surface high-impact contributors.
+
+### Goal 3 — Drive Community Engagement & Contribution
+
+**Q2 Activities**
+
+- Increased social posting cadence (LinkedIn: 78 posts, +63%)
+
+- Active Slack channels around events
+
+**Evidence**
+
+- LinkedIn impressions +104%, clicks +101%, engagement rate 8.06%
+
+- Slack growth +9.4%; newsletter subscriber growth +7.3%.
+
+**Next steps**
+
+- Scale the Student Ambassador Program promotion and onboarding (Q3 focus)
+
+- Implement quarterly contributor surveys to close feedback loops.
+
+**Recomendation**
+
+- Targeted email campaigns
+
+- AMAs and interactive webinars.
+
+## Marketing Working Group Performance Overview
+
+### Social Media & Community Performance
+
+| **Platform** | **Start of Q2** | **End of Q2** | **% Change** | **Key Insights**|
+|---------|-------------|-----------|-----------|-------------|
+LinkedIn | ~4,249 | 4,534 | +7% | Strong post cadence (78 posts), impressions doubled.click-throughs and engagement increased significantly
```

Comment:
please improve columns description to explain what the numbers replresent, for example what `4,249` is, is it impression, views, engagement or what exactly